tests/testthat/setup.R

```diff
@@ -1,13 +1,14 @@
 library(mlr3)
 attachNamespace("checkmate")
-old_opts = options(
-  warnPartialMatchArgs = TRUE,
-  warnPartialMatchAttr = TRUE,
-  warnPartialMatchDollar = TRUE
-)
+# The upstream package causes partial match warnings
```

Comment:
Which one?

Reply:
`irace` uses `rep` with `length` instead of `length.out`

Reply:
Reported upstream https://github.com/MLopez-Ibanez/irace/issues/7.

Reply:
For what is worth, this issue is now fixed upstream so you should be able to re-enable the warnings. We, the irace team (which is basically 2 people doing both teaching and research), are happy to receive issues and pull requests for fixes like these but also for changes that simplify integration with mlr (or other packages).